package.json

```diff
@@ -31,6 +33,8 @@
     "turbo": "^1.7.0"
   },
   "dependencies": {
+    "@changesets/cli": "^2.26.0",
     "instantsearch.css": "^8.0.0"
-  }
+  },
+  "version": "0.0.0"
```

Comment:
Can you move to the top? 😄 Also, 0.0.1 O think is more appropriate (even if is going to be eternal) :)

Reply:
I added `0.0.0` as this is what seems to be the convention in the doc and the repo examples I came across.
- https://turbo.build/repo/docs/reference/codemods#add-package-manager